query-service-impl/src/test/java/org/hypertrace/core/query/service/postgres/QueryRequestToPostgresSQLConverterTest.java

```diff
@@ -307,7 +307,7 @@ void testQueryWithOrderBy() {
             + " = '"
             + TENANT_ID
             + "' "
-            + "order by start_time_millis desc , end_time_millis limit 100",
+            + "order by 2 desc, 3 limit 100",
```

Comment:
Any specific reason to move to ordered number instead of explicit names?

Reply:
As we discussed, it is done to avoid any function call repetition between select and order-by/group-b